### PR TITLE
[DC-1521] Shared Sandbox Name Fix

### PR DESCRIPTION
**Pull request by dcampbell-vumc**
* Updated `create_tier.create_datasets` to drop the tier from the sandbox_dataset_id
* Updated `create_tier_test.test_create_datasets` to test the changes
  made in `create_tier.create_datasets`

### CODE REVIEW

File: tests/unit_tests/data_steward/tools/create_tier_test.py [hunk 1]
```diff
@@ -288,26 +288,30 @@ def test_create_dataset(self, mock_client, mock_define_dataset,
...
+                          self.dataset_name, self.input_dataset, None,
+                          self.release_tag)
+        self.assertRaises(RuntimeError, create_datasets, client,
+                          self.dataset_name, self.input_dataset, self.tier,
+                          None)
 
         # Test
-        expected = create_datasets(client, self.name, self.input_dataset,
-                                   self.tier, self.release_tag)
+        expected = create_datasets(client, self.dataset_name,
```
Comment (jp3477) [inline tests/unit_tests/data_steward/tools/create_tier_test.py:312]: NIT: this variable should probably be called actual

**Review by jp3477 — CHANGES_REQUESTED** [review]
Would you be able to repeat the same test case for a registered tier dataset? May be redundant, but I think it's in the ticket acceptance criteria. Other than that, everything looks great.

File: tests/unit_tests/data_steward/tools/create_tier_test.py (hunk 1 above)
Comment (dcampbell-vumc) [reply tests/unit_tests/data_steward/tools/create_tier_test.py:312]: You're right! I changed it

**Review by jp3477 — APPROVED** [review]
Great, approved!

**Pull request merged**